src/modules/coreHttpApi/openapi.yml

```
@@ -2130,8 +2130,8 @@ paths:
               - $ref: "#/components/schemas/FileID"
               - $ref: "#/components/schemas/FileReferenceTruncated"
         - in: query
-          name: newQRCodeFormat
-          description: If set to true, the QRCode will contain the new url format. This is only relevant if the accept header is set to `image/png`.
+          name: oldQRCodeFormat
+          description: If set to true, the QRCode will contain the old url format. This is only relevant if the accept header is set to `image/png`. This option is available to provide a grace period for all apps to support the new QR code format and will be removed in the future.
```

Just as you write `"QR code"` in your added sentence, please also change `"QRCode"` to `"QR code"` in the first sentence.

The same applies to the other descriptions.

```suggestion
          description: If set to true, the QR code will contain the old url format. This is only relevant if the accept header is set to `image/png`. This option is available to provide a grace period for all apps to support the new QR code format and will be removed in the future.
```

---

As I removed that backwards compatibility this is now outdated :)